chart/values.yaml

```diff
@@ -79,8 +79,8 @@ images:
     tag: 0.11.0
     pullPolicy: IfNotPresent
   redis:
-    repository: redis
-    tag: 6-buster
+    repository: bitnami/redis
```

Comment:
@gardnerdev may you provided more detailed clarification for this change? Otherwise it doesn't seem a good idea to deviate away from the default "official" image.

Reply:
Sure, bitnami/redis is working as non-root process which is following best practices. When you are sharing cluster (namespace e.g) you are not allowed to run root-processes. Please, refer to https://docs.bitnami.com/tutorials/work-with-non-root-containers/

Reply:
Personally I still don't think it's a good idea to deviate away from the official Docker image.

Regarding the necessarily to run containers as non-root in K8S, I think something like K8S [Security Context](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/) already suffices.

Reply:
Not really, security context will not work, because of lack of specific users. Bitnami images are widely used and batter than "stable" imo

Reply:
I would leave it for comments from other committers as well first. But in my view I would give a -1 for now.

Reply:
The official docker image already runs as non-root user by default (50000 - Airflow) - precisely as explained by @gardnerdev. 

It also has another feature - the image is OpenShift compatible: https://github.com/apache/airflow/issues/9248 which means that it can essentially run as "random" user but the user has to belong to "root" group (this is how OpenShift runs the images).

I think if the bitnami image is also OpenShift compatible - I'd definitely welcome that!